frontend/src/components/Pagination.scss

```diff
@@ -0,0 +1,36 @@
+.pagination {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 30px;
+ }
+ 
+ .prevNextItem {
+  color: #005ea2;
+  cursor: pointer;
+ }
+
+ .dots {
+  color: #005ea2;
+ }
+
+ .paginationItem {
+  height: 40px;
+  width: 40px;
+  color: #005ea2;
```

Comment:
This color needs to be updated. We are not using any blue in the theme so far.
For reference use this:
https://designsystem.digital.gov/utilities/color/

Reply:
Fixed